### PR TITLE
Bulk user upload e2e tests

### PR DESCRIPTION
**Pull request by njogz**
# Description

This PR adds e2e tests for the bulk user upload feature

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: admin/src/js/services/location.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ angular.module('inboxServices').factory('Location',
     'ngInject';
 
     const location = $window.location;
-    const dbName = 'medic';
+    const dbName = $window.localStorage.getItem('dbName') || 'medic';
```
Comment (latin-panda) [inline admin/src/js/services/location.js:8]: For db kind of things I prefer to be more restrictive, many things can happen with a wrong db:
```
const isTestEnv = $window.localStorage.getItem('isTestEnv');
const dbName = isTestEnv ? 'medic-test' : 'medic';
```

File: tests/e2e/users/bulk-user-upload.wdio-spec.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const loginPage = require('../../page-objects/login/login.wdio.page');
+
+describe('Bulk User Creation ->', () => {
+  before(async () => {
+    await loginPage.cookieLogin();
+  });
+
+  beforeEach(async () => {
+    await browser.execute(function() {
+      this.localStorage.setItem('dbName', 'medic-test');
```
Comment (latin-panda) [inline tests/e2e/users/bulk-user-upload.wdio-spec.js:11]: Therefore here is:
```suggestion
      this.localStorage.setItem('isTestEnv', true);
```

File: tests/page-objects/admin/user.wdio.page.js [hunk 3]
```diff
@@ -119,6 +136,35 @@ const getContactErrorText = async () => {
...
+
+const openUploadUsersDialog = async () => {
+  await (await uploadUsersButton()).waitForDisplayed();
+  await (await uploadUsersButton()).click();
+  await (await uploadUsersDialog()).waitForDisplayed();
+  // wait for animations to finish
+  await browser.pause(500);
+};
+
+const uploadSummary = async () => {
```
Comment (latin-panda) [inline tests/page-objects/admin/user.wdio.page.js:164]: I'd think this function uploads summary, I think this name is clearer:
```suggestion
const waitForUploadSummary = async () => {
```

**Review by latin-panda — CHANGES_REQUESTED** [review]
Nice! Happy to see this test, I've left some comments in line

**Comment by njogz** [comment]
@latin-panda I have addressed your feedback. Please have another look.

**Review by latin-panda — APPROVED** [review]
Thank you!

**Pull request merged**